index.html

```diff
@@ -9,118 +9,116 @@
           photoshop photo editing skills.
-          <br />
+        </p>
+        <p>
           Over the past year I have moved more into learning and exploring the
           coding side of eCommerce websites, and am very interested in learning
           more about front end web and back end web development.
         </p>
       </section>
 
       <section id="skills">
-        <header>
-          <h4>skills</h4>
-        </header>
+        <h4>Skills</h4>
         <ul></ul>
       </section>
 
-      <section id="connect">
-        <header>
-          <h2>let's connect</h2>
-        </header>
-
-        <nav>
-          <ul class="connect">
-            <li>
-              <a
-                href="mailto:keriwest88@gmail.com?subject=Mail from Website"
-                target="_blank"
-                >Email</a
-              >
-            </li>
-            <li>
-              <a href="https://github.com/wessstt" target="_blank">GitHub</a>
-            </li>
-            <li>
-              <a href="https://www.linkedin.com/in/keri-west/" target="_blank"
-                >LinkedIn</a
-              >
-            </li>
-          </ul>
-        </nav>
-      </section>
-
       <section id="form" class="position">
         <form id="form" name="leave_message">
-          <label id="usersName" for="name">name</label>
-          <input id="name" type="text" name="usersName" required="true" />
+          <h2>Let's chat</h2>
+          <fieldset id="userInput">
+            <label id="usersName" for="name">Name</label>
+            <input id="name" type="text" name="usersName" required="true" />
+            <label id="usersEmail" for="email">Email</label>
+            <input id="email" type="email" name="usersEmail" required="true" />
+          </fieldset>
+          <fieldset id="userMessage">
+            <label id="userMessage" for="message">Message</label>
+            <textarea
+              id="message"
+              name="usersMessage"
+              required="true"
+            ></textarea>
+          </fieldset>
           <br />
-          <label id="usersEmail" for="email">email</label>
-          <input id="email" type="email" name="usersEmail" required="true" />
-
-          <label id="userMessage" for="message">message</label>
-          <textarea id="message" name="usersMessage" required="true"></textarea>
-          <br />
-          <button type="submit">submit</button>
+          <button type="submit">Submit</button>
         </form>
       </section>
 
       <section id="messages">
-        <h2 id="msgHeader">messages</h2>
+        <h2>Messages</h2>
         <ul></ul>
       </section>
 
-      <footer></footer>
+      <footer>
+        <section id="connect">
+          <nav id="connect">
+            <a
+              href="mailto:keriwest88@gmail.com?subject=Mail from Website"
+              target="_blank"
+            >
+              <img src="img/email-icon.png"
+            /></a>
```

Comment:
```suggestion
            ><img src="img/email-icon.png"></a>
```

You don't need the `/>` in HTML5. HTML5 is not XML, where closing tags are always required. https://developer.mozilla.org/en-US/docs/Glossary/Void_element